welcome.js

```diff
@@ -1,14 +1,4 @@
 /* eslint-disable no-console */
-console.log();
-console.log('*************************************************************************');
-console.log('*                                                                       *');
-console.log('* To get started, please create a twilio-cli profile:                   *');
-console.log('*                                                                       *');
-console.log('*     twilio login                                                      *');
-console.log('*                                                                       *');
-console.log('*     OR                                                                *');
-console.log('*                                                                       *');
-console.log('*     twilio profiles:create                                            *');
-console.log('*                                                                       *');
-console.log('*************************************************************************');
-console.log();
+const { PostInstallDisplayManager } = require('@twilio/cli-core').postInstallUtility;
```

Comment:
Any reason why this should be in core and not this repo? I don't expect plugins to utilize this functionality.

Reply:
this is so we have more control over the upgrade alerting no matter the entry point like npm, or homebrew, or anything else we might support in the future

Reply:
Yeah, but those are all for twilio-cli installations. Core is intended to provide common functionality for plugins to utilize. I tend to lean towards only putting things in core if we expect to support plugin use cases.

Reply:
Ok, gotcha. It makes sense to keep that here then

Reply:
Moved `post-install-utility` from core to this package.